_topic_map.yml

```diff
@@ -56,12 +56,14 @@ Topics:
 - Name: Networking
   File: dedicated-networking
 ---
-  Name: Understanding ROSA
-  Dir: understanding_rosa
+  Name: Product Overview
```

Comment:
Sentence case.

Reply:
Done. It's gonna take me awhile to get used to sentence case...